src/howler.core.js

```diff
@@ -2353,6 +2355,7 @@
     } else {
       // Load the buffer from the URL.
       var xhr = new XMLHttpRequest();
+      Howler.xhr = xhr;
```

Comment:
What do we achieve by this?

Reply:
What I write in the description. Improved error handling, progress and ability to abort request. I think we should consider implementing some of this in howler and contributing back, but this makes it possible to reuse some of the logic from the old audio manager (as it seemed the retrying and handling of loading worked pretty well there).

Reply:
I see the point after looking at the player PR. :-)